src/main/java/duke/parser/Action.java

```diff
@@ -0,0 +1,25 @@
+package duke.parser;
+
+enum Action {
+    DEADLINE("deadline"), DELETE("delete"), ERROR("error"), EVENT("event"),
+    LIST("list"), MARK("mark"), TODO("todo"), UNMARK("unmark"), FIND("find");
+
+    private final String action;
+
+    Action(String action) {
+        this.action = action;
+    }
+
+    public String getAction() {
+        return action;
+    }
+
+    public static Action getAction(String action) {
```

Comment:
Could you describe what this method is supposed to do?

Reply:
Umm it's just a getter, to my knowledge getters and setters don't need explicit JavaDocs.